content/articles/2018-12-textrecipes-0-0-1.Rmarkdown

```diff
@@ -0,0 +1,84 @@
+---
+title: 'textrecipes 0.0.1'
+author: Emil Hvitfeldt
+date: '2018-12-20'
+slug: textrecipes-0-0-1
+description: >
+    textrecipes 0.0.1 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/X1exjxxBho4
+  author: Roman Kraft
+---
+
+```{r setup, include = FALSE}
+library(textrecipes)
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're delighted to announce the release of [textrecipes 0.0.1](https://github.com/tidymodels/textrecipes) on CRAN. [`textrecipes`](https://github.com/tidymodels/textrecipes) implements a collection of new steps for the [`recipes`](https://github.com/tidymodels/recipes) package to deal with text preprocessing. [`textrecipes`](https://github.com/tidymodels/textrecipes) is still in early development so any and all feedback is highly appreciated.
```

Comment:
Does textrecipes have a pkgdown site yet? (Oh, it does: https://tidymodels.github.io/textrecipes/ )

I'd link to the pkgdown site for at least one of the mentions of textrecipes (currently you're going to the GitHub repo three times), and get rid of the repeat after your first use of a link.